modules/Hoot/tools/overpassQueryPanel.js

```diff
@@ -122,11 +124,57 @@ export default class OverpassQueryPanel {
                 }
             });
 
+        this.buildQueryOptions();
+
         const errorInfoContainer = this.form.select( '.hoot-menu' )
             .insert( 'div', '.modal-footer' )
             .classed( 'badData', true );
     }
 
+    buildQueryOptions() {
+        const options = [ 'Buildings', 'Highways' ];
+
+        this.queryOptions = this.overpassQueryContainer.append( 'div' )
+            .classed( 'queryBuilder hidden', true )
+            .text( 'Build query options' );
+
+        options.forEach( option => {
+            this.queryOptions.append( 'label' )
+                .text( option )
+                .append( 'input' )
+                .attr( 'id', option )
+                .attr( 'type', 'checkbox' )
+                .on( 'click', () => {
+                    this.queryBuilder();
+                } );
+        } );
```

Comment:
Could this be done the d3 way with `.data().enter().append()`?
Then the bound data could be used to drive the queryBuilder below by retrieving a tag term in a selection.
It should also clear out the query if no boxes are checked.